### DISCUSSION TITLE
Withdrawal

### DISCUSSION THREAD

**Discussion by Mategio14** [Chia-Network/chia-blockchain#17221] [Support]
Hello folks. Again. Transaction  is pending 5th day. Please. somebody will. help me?

**Comment by BrandtH22** [comment]
Hey @Mategio14 , as a heads up we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia).

The chain has seen high utilization leading to fee pressure so any transactions submitted without fees are taking lower priority than those which have fees. Many users have been able to resolve their pending fees by cancelling the previous transaction onchain and resubmitting it with a higher fee. 

We have created this guide of standard transaction types and their minimum effective fees (minimum meaning that time sensitive transactions may require more in fees to be processed accordingly):
https://docs.chia.net/coin-set-costs/#estimated-transaction-costs

Let us know if you have any questions or run into issues when resubmitting the transaction :+1:

**Comment by Mategio14** [comment]
Thanks for quick response Clyde. One more question. I am on main Chia wallet and how to can sell this transaction? Just delete unsecsecuful transaction? Thanks 

Sent from Yahoo Mail on Android 
 
  On Tue, 9 Jan 2024 at 15:44, ***@***.***> wrote:   


Hey @Mategio14 , as a heads up we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia).

The chain has seen high utilization leading to fee pressure so any transactions submitted without fees are taking lower priority than those which have fees. Many users have been able to resolve their pending fees by cancelling the previous transaction onchain and resubmitting it with a higher fee.

We have created this guide of standard transaction types and their minimum effective fees (minimum meaning that time sensitive transactions may require more in fees to be processed accordingly):
https://docs.chia.net/coin-set-costs/#estimated-transaction-costs

Let us know if you have any questions or run into issues when resubmitting the transaction 👍

**Reply by BrandtH22** [reply]
Hey @Mategio14 , the thing to remember about the delete unconfirmed transactions is that it only effects your local machine and not the mempool at large. This means that just deleting unconfirmed can still lead to a duplicate transaction if the resubmission ends up using different coin IDs than the original transaction. 

Depending on the amount you are sending compared to your balance you may first need to effectively cancel the transaction on-chain. 

My rule of thumb is if my initial transaction is less than 70% of my total balance I will:
1. stop chia
2. rename the wallet db at `~\.chia\mainnet\wallet\db\`
3. start chia and let it sync from scratch (this removes the wallets record of the previous transaction)
4. send the full wallet balance to self with a reasonable fee (note that if you have a lot of highly fragmented coins then you may need to send parts of the balance at a time, this can be seen by an error similar to "Can only send x mojos in a block, y was received").
5. Once that completes then reattempt the original transaction with a reasonable fee
6. Optional - if you had offer files or cat/did names in the original wallet file you may want to stop chia, name the wallet db back (deleting the newly created one), and using the gui settings / advanced / resync option. This will resync the wallet with the offer file and cat/did information

If my initial transaction is more than 70% or I do not mind there being a potential duplicate transaction (generally only when the initial transaction was being sent to one of my other wallets) then I will perform the same steps as above but skip number 4.

Let us know if you run into any issues or questions during the process and take note that we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia)

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
